server/models/CodeLog.coffee

```diff
@@ -0,0 +1,24 @@
+mongoose = require 'mongoose'
+config = require '../../server_config'
+
+CodeLogSchema = new mongoose.Schema({
+  created:
+    type: Date
+    'default': Date.now
+}, {strict: false, read: config.mongo.readpref})
```

Comment:
You should add userID and sessionID to the schema here. This will make Mongoose automagically convert strings ids to MongoDB ObjectIds.